html_css/v2/main-course/responsive-css/responsive-css-introduction.md

```diff
@@ -0,0 +1,26 @@
+### Introduction
+"Responsive Design" is an incredibly important concept when it comes to developing for the web. In many ways you are already equipped to create beautiful responsive websites. In this section we're going to fill in the gaps.
+
+### Learning Outcomes
+* You'll learn what 'Responsive Design' is and why it's important.
+
+### Terminology
+The phrase 'Responsive Design' is the most common phrase across the internet to refer to the concept we're talking about here. Unfortunately, it is a little bit misleading, because of the word 'design'. These lessons are not going to focus much on the actual _design_ of responsive websites, but the techniques you can use to implement responsiveness on websites.
+
+The word 'design' refers to making decisions about what kinds of things look nice, crafting a user experience and deciding exactly what a project should look like. There might be some tidbits about that sprinkled throughout these lessons, but it is not the primary focus. HOWEVER, since this phrase has been pretty much universally adopted, it's the phrase we'll use here.
+
+### So what _IS_ it?
+Responsive design is creating websites that work correctly on any screen. Basically since the release of the first iPhone in 2007, it has become a _requirement_ that your web projects function correctly on everything from normal desktop monitors to tiny phone screens. 
+
+As you have likely discovered, this kind of responsiveness doesn't often happen automatically, especially once your projects get more complex than simple text on a page. (Your first HTML-only recipe project probably worked just fine on a phone, aside from the images getting cut-off.) So, when it comes down to it, "Responsive Web Design" is a set of techniques you can use to make your sites work on any size screen. 
```

Comment:
Parentheses are unnecessary, you can make that another full sentence. Or, if you want to keep them, they should come before the period of the previous sentence since you are elaborating on the point you made there.

No hyphen in "cut off"